src/components/Navigation.tsx

```diff
@@ -0,0 +1,32 @@
+import { NavLink } from 'react-router-dom';
+
+export const Navigation = () => {
+  const getActive = (isActive:boolean) => `navbar-item ${isActive && 'has-background-grey-lighter'}`;
```

Comment:
```suggestion
  const getActiveClassName = (isActive:boolean) => `navbar-item ${isActive && 'has-background-grey-lighter'}`;
```

let's use more informative naming